src/components/Quote.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+class Quote extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      message: 'The will of man is his happiness. -Friedrich Schiller',
+    };
+  }
+
+  render() {
+    const { message } = this.state;
+    return (
+      <div className="quote">
+        <h1>Quote</h1>
+        <div className="content">
+          <p>{message}</p>
+        </div>
+      </div>
+    );
+  }
+}
+
+export default Quote;
```

Comment:
* As part of the requirement, you are supposed to get quotes from the external API [API Ninjas](https://api-ninjas.com/api/quotes) and not by hardcoding it.
* Ensure all the project requirements are met.
![Screenshot from 2023-06-02 21-26-36](https://github.com/Enoisong/math-magicians/assets/44624138/f87eefcc-68cc-4461-b869-284beca8858f)